pkg/operation/botanist/controlplane.go

```diff
@@ -44,6 +45,7 @@ func (b *Botanist) DeployNamespace() error {
 			Name:        b.Shoot.SeedNamespace,
 			Labels: map[string]string{
 				common.GardenRole: common.GardenRoleShoot,
+				"hibernated":      strconv.FormatBool(b.Shoot.IsHibernated),
```

Comment:
I would prefer the key to be prefixed with something, for example `"shoot.sapcloud.io/hibernated"`

Also, this should be annotation, just like we are doing for `custom.shoot.sapcloud.io/.*` annotations of Shoot resources. Of course, we could keep both, the annotation and label, because `Validating` and `Mutating` webhooks could filter resources based on the [namespace labels](https://github.com/kubernetes/kubernetes/blob/v1.13.0/staging/src/k8s.io/api/admissionregistration/v1beta1/types.go#L209). In this way the admission webhooks could limit their work only to hibernated or non-hibernated shoots. 

What do you think about using both annotation and label ?

Reply:
No strong opinion. This is just a gap filler until we have all extension resources. They will have `.spec.hibernated` indicating whether the shoot is hibernated or not, so the label/annotation will go away anyways in the future.
I can rename it to `shoot.garden.sapcloud.io/hibernated`, sure. AFAIK, you are the only user of the annotations/labels on the namespace object. So tell me what you prefer (annotation vs. label).

Reply:
Honestly, I prefer both, but in reality it is not necessary to support both of them.
With that said and the fact that we could benefit from label selectors, let it be label, not annotation.